src/hotspot/share/jfr/recorder/checkpoint/types/jfrType.cpp

```diff
@@ -214,7 +214,7 @@ void MetaspaceObjectTypeConstant::serialize(JfrCheckpointWriter& writer) {
 static const char* reference_type_to_string(ReferenceType rt) {
   switch (rt) {
     case REF_NONE: return "None reference";
-    case REF_OTHER: return "Other reference";
+    case REF_REFERENCE: return "j.l.r.Reference";
```

Comment:
I think either "Reference" or "java.lang.ref.Reference" would be better than the abbreviation.